.github/workflows/ci.yaml

```diff
@@ -133,6 +133,7 @@ jobs:
           -DROCM_ENABLE_GH_ANNOTATIONS=On \
           -DCLANG_TIDY_DEPEND_ON_TARGET=Off \
           -DCLANG_TIDY_CACHE=/data/tidy-cache \
+          -DGPU_TARGETS=gfx908 \
```

Comment:
Does this mean ROCM-Ubuntu server has gfx908 ? 
I thought it was CPU only server.

Reply:
I think its needed for this issue with targets.

Reply:
No, it doesnt have any gpu, but we need to build for at least one target to get build/tidy coverage. We could add multiple targets to this as well.